vars/Debian.yml

```diff
@@ -0,0 +1,4 @@
+---
+i2_user: nagios
```

Comment:
Excuse me ???

Reply:
On Debian the `nagios` user is used for historical reasons, so this should be fine.

Reply:
yup, `nagios` is default on debian